frontend/icons/content-modules-wide.svg

```diff
@@ -0,0 +1,14 @@
+<svg id="content-modules-wide" width="33px" height="32px" viewBox="0 0 33 32" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
All other icons are `<symbols>` and please pipe this one through [SVGOMG](https://jakearchibald.github.io/svgomg/).